pkg/assets/assets_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package assets_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/kinvolk/lokomotive/pkg/assets"
+)
+
+// Verify assets are successfully extracted from memory to disk.
+func TestExtract(t *testing.T) {
+	dir, err := ioutil.TempDir("", "lokoctl")
+	if err != nil {
+		t.Fatalf("Creating temp dir: %v", err)
```

Comment:
Test cases should not be capitalized the same as returned errors:
```suggestion
		t.Fatalf("creating temp dir: %v", err)
```

Reply:
Not sure what you mean regarding test cases, but regarding the error string, IMO the **beginning** of an error (namely the place where you call `Printf/Logf/Fatalf`) *should* be capitalized because it's the beginning of the text the user sees. 

As far as I understand the "do not capitalize" convention applies to errors you **return** (because they will likely be appended to existing error strings).

We are currently inconsistent about this in our codebase, and I'd rather us settle on capitalizing the beginning of error strings.

:ok_hand: 

```
Deploying cluster: rendering assets: doing this and that: no such file or directory
```

:confused: 

```
deploying cluster: rendering assets: doing this and that: no such file or directory
```

Reply:
So it would be:
```sh
$ go test ./
--- FAIL: TestHelloWorld (0.00s)
    main_test.go:6: not implemented
FAIL
FAIL    foo     0.001s
FAIL
```

The `not implemented` is after `:`, which gives me a feeling, that it should not be capitalized, as it's the Go testing framework controlling the output, not us. (In case of top-level logging, I agree that they should be capitalized).

Reply:
IMO it makes sense to have one standard for error strings. An error string is an error string, doesn't matter if it's logged, printed or handled by the testing package.

Anyway, we are currently inconsistent. I can change all occurrences in this PR but I suggest you open a new issue to track this if you want this changed in general.